src/CMakeLists.txt

```diff
@@ -175,6 +178,19 @@ if(LIBRW_PLATFORM_GL3)
             )
         endif()
     endif()
+
+    if(NINTENDO_SWITCH)
+        find_library(SWITCH_EGL_LIBRARY EGL)
```

Comment:
I forgot about this pr.
I will still create a pr on your fork with various changes to make the generated tarball relocatable (=movable between systems).